subprojects/code-quality/src/main/groovy/org/gradle/api/plugins/quality/Checkstyle.java

```diff
@@ -364,19 +364,7 @@ public void setIgnoreFailures(boolean ignoreFailures) {
      * @since 3.4
```

Comment:
🤔 What should happen with the `@since` annotations? The method is not really present since 3.4, but we will keep it functioning as if it was...